docs/machine-learning/resources/learners.md

```diff
@@ -0,0 +1,67 @@
+# Learning Algorithms
+The following table will help you understand the different learning algorithms (learners) included in ML.NET and the [machine learning tasks](tasks.md) that they can be used for. Click on the learner name for more in-depth explanations on each of these learning methods.
+
+> [!NOTE]
+> ML.NET is currently in Preview. Not all learning algorithms are currently supported. To submit a request for a certain learner, please open an issue in the [dotnet/machinelearning repository](https://github.com/dotnet/machinelearning/issues).
+
+| Learner Name | Binary classification | Multi-class classification | Regression | Ranking | Clustering | Anomaly Detection |
```

Comment:
In other docs we are spelling "Multiclass", not  "Multi-class". I'd suggest to be consistent. in ML world "Multiclass" seems to be more common (https://en.wikipedia.org/wiki/Multiclass_classification, http://www.mit.edu/~9.520/spring09/Classes/multiclass.pdf, https://www.coursera.org/learn/machine-learning/lecture/gFpiW/multiclass-classification) .